src/log-httplog.c

```diff
@@ -709,6 +709,12 @@ static void LogHttpLogDeInitCtx(OutputCtx *output_ctx)
 {
     LogHttpFileCtx *httplog_ctx = (LogHttpFileCtx *)output_ctx->data;
     uint32_t i;
+
+    if (httplog_ctx->file_ctx->dropped) {
```

Comment:
Yes, so the counter is generalized, its just the logging.

Might be kind of hard as the code calling ogHttpLogDeInitCtx (or other DeInitCtx) doesn't know if they are using common writer or not, the calling code just has an opage void data pointer to logger specific data.

What might be more useful is a stat counter? Either per logger, or a simpler single counter for dropped log messages?